src/test/fuzz/addrdb.cpp

```diff
@@ -32,4 +34,5 @@ void test_one_input(const std::vector<uint8_t>& buffer)
         }
         return CBanEntry{};
     }();
+    (void)ban_entry;
```

Comment:
and then comment this as 
```suggestion
    (void)ban_entry; // currently unused
```